### PR TITLE
First stab at docs

### PR DESCRIPTION
**Pull request by sihugh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,43 @@
...
+Generally you might want to:
+ - amend existing lines
+ - reorder links within sections
+ - add accordion sections
+
+Once you've finished your edits, you'll need to commit the changes and make a
+pull request.  (You can ask the on-call dev to help you with this if you like).
+
+You then need to get your pull request approved and merged by someone else.
+GOV.UK developers and other members of the GOV.UK Content Design Github team
```
Comment (issyl0) [inline README.md:33]:
```suggestion
GOV.UK developers and other members of the GOV.UK Content Design GitHub team
```

File: README.md [hunk 2]
```diff
@@ -1 +1,43 @@
...
+
+## Live examples
+
+https://www.gov.uk/coronavirus
+
+## Editing the content as a content designer
+
+You will need to be a member of the GOV.UK content designers Github group. You
+may need to raise a Zendesk ticket pointing 2nd Line at the [instructions for
+adding a content designer to Github](https://docs.publishing.service.gov.uk/manual/give-a-content-designer-access-to-github.html).
```
Comment (issyl0) [inline README.md:13]:
```suggestion
adding a content designer to GitHub](https://docs.publishing.service.gov.uk/manual/give-a-content-designer-access-to-github.html).
```

File: README.md [hunk 3]
```diff
@@ -1 +1,43 @@
...
+
+Stores the content for the www.gov.uk/coronavirus page.
+
+## Live examples
+
+https://www.gov.uk/coronavirus
+
+## Editing the content as a content designer
+
+You will need to be a member of the GOV.UK content designers Github group. You
```
Comment (issyl0) [inline README.md:11]:
```suggestion
You will need to be a member of the GOV.UK content designers GitHub team. You
```

**Comment by hannako** [comment]
@issyl0 Was the branding comment about the github typos? I've fixed those, could you approve?

**Pull request merged**